release.sh

```diff
@@ -99,8 +99,7 @@ cd $vdir
 # Get linux 20.04 artifact ==========================
 
 echo "Looking up linux 20.04 artifact ..."
-
-dlurl=`curl -s -H "Authorization: token $ghtoken" -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kadena-io/pact/actions/runs/$runid/artifacts | jq -r '.artifacts[] | select(.name | contains ("8.10.7.ubuntu-20.04")) | .archive_download_url'`
+dlurl=`curl -s -H "Authorization: token $ghtoken" -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kadena-io/pact/actions/runs/$runid/artifacts | jq -r '.artifacts[] | select(.name | contains ("9.6.ubuntu-20.04")) | .archive_download_url'`
```

Comment:
Is it just 9.6 and not 9.6.3 or 9.6.2?

Reply:
Yeah, there's no qualifier present in the ci: https://github.com/kadena-io/pact/blob/39737082235067d839e42cd023a50e0859340f99/.github/workflows/applications.yml#L16. This is brittle - I'm going to work with @DevopsGoth to migrate away from these static strings in our release script to some thing a little more robust